src/App.js

```diff
@@ -17,6 +18,7 @@ export default function App() {
         <Route exact path="/" component={Mainpage} />
         <Route path="/agissons" component={Agissons} />
         <Route path="/connexion" component={Connexion} />
+    <Route path="/inscription" component={Inscription} />
```

Comment:
**Critical**
Indentation failure -> Netlify was not used -> PR was not read :cry: